plop/hooks/hook-story.txt

```diff
@@ -20,16 +19,28 @@ Please write here the component description
   <Story name="Overview">
     {() => {
       const ref = useRef(null);
-      {{camelCase componentName}}({ ref, callback });
+      const { result } = {{camelCase hookName}}({ ref });
       return (
-        <div ref={ref}>
-          Reference Element
+        <div className={styles.container}>
+          Random hook result = {result}
         </div>
       );
     }}
   </Story>
 </Canvas>
 
-## Props
-<ArgsTable of={ {{properCase componentName}} } />
+## Arguments
 
+<FunctionArguments>
+  <FunctionArgument name="options" type="Object">
+    <FunctionArgument name="value" type="any" description={<>Example of required argument.</>} required />
```

Comment:
I am not sure <> is needed if the value of the description is text only. it does not working with string?

Reply:
and maybe let's add here another fields example such as string or int - inside the options?

Reply:
It doesn't work with a string